tika-parsers/tika-parsers-standard/tika-parsers-standard-modules/tika-parser-microsoft-module/src/main/java/org/apache/tika/parser/microsoft/onenote/FileNode.java

```diff
@@ -257,11 +257,11 @@ public void print(OneNoteDocument document, OneNotePtr pointer, int indentLevel)
                     subType.revisionManifest.revisionRole);
 
         }
-        if ((gctxid != ExtendedGUID.nil() ||
+        if ((!gctxid.equals(ExtendedGUID.nil()) ||
```

Comment:
Good catch!  We should probably make a static constant ExtendedGUID.NIL to avoid unnecessary object creation.

Reply:
To be clear, I'm not asking you to do the static thing on this issue.  Your catch is important.  Thank you!

Reply:
You are welcome!